### DISCUSSION TITLE
Preview type issue...

### DISCUSSION THREAD

**Discussion by Duckinm** [payloadcms/payload#280] [General]
I did get this type of error, How to prevent or avoid it?

![Screen Shot 2564-08-11 at 10 12 13 AM](https://user-images.githubusercontent.com/29366271/128964017-9dfaf6d9-7a20-4d62-9c7b-533141682d2e.png)

![Screen Shot 2564-08-11 at 10 15 35 AM](https://user-images.githubusercontent.com/29366271/128964081-adb26695-e8f4-4a7e-a174-fd866898a608.png)

**Comment by jmikrut** [comment]
Hey @Duckinm I can't recreate this!

What version of Payload are you running? This exact code works in a few of our apps. You may need to re-yarn, or try and update Payload?
